rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -35,6 +35,8 @@ Requires: %{?scl_prefix}rubygem(git)
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: foreman-plugin >= 1.15
```

Comment:
this is not related to apipie but according to gem2rpm template it seems it should be here unconditionally